src/scss/components/_ratings.scss

```diff
@@ -0,0 +1,63 @@
+$star-count: 6;
+$star-size-small: 14px;
+$star-size-medium: 28px;
+$star-size-large: 42px;
+
+.c-ratings {
+    background: inline('#{$icon-path}/stars/star--empty.svg') repeat-x;
+    background-size: contain;
+    display: inline-block;
+    height: $star-size-small;
+    width: $star-size-small * $star-count;
+}
+
+.c-ratings--medium {
+    height: $star-size-medium;
+    width: $star-size-medium * $star-count;
+}
+
+.c-ratings--large {
+    height: $star-size-large;
+    width: $star-size-large * $star-count;
+}
+
+.c-ratings--fill {
+    background: inline('#{$icon-path}/stars/star--filled.svg') repeat-x;
+    background-size: contain;
+    display: inline-block;
+    height: 100%;
+    width: 100%;
+    @include rating-fill($star-count);
+}
+
+/*
+Selectable stars rating component
+*/
+.c-ratingSelectable {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+}
+
+.c-ratingSelectable-input {
+    display: none;
+
+    &:checked {
+        & + label,
+        & ~ input + label {
+           background-image: inline('#{$icon-path}/stars/star--filled.svg');
```

Comment:
Would make sense to make a silent extends class for the icons.  At the moment, if the icon name for filled stars changed, it would need to be changed in 3 places.  Would be better if it was one place 🔁

Reply:
It probably makes sense to be a part of `f-icons` as I would think it makes most sense to call the classes `%c-icon--star--fill` and ``%c-icon--star--empty`.